src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="form"
+    method="post"
+    data-qa="big"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
```

Comment:
```suggestion
    <form
      class="form"
      method="post"
      data-qa="big"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
    >
```
Добавляй отступы атрибутам